docs/proposals/pause-ensure-work/README.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Pause ensure work
+authors:
+- "@CharlesQQ"
+reviewers:
+- "@RainbowMango"
+approvers:
+
+creation-date: 2022-05-21
+
+---
+
+# Pause ensure work
+
+## Summary
+
+When the resource (as known as `resource template`) paused field(such as label `work.karmada.io/paused`)  is true, we hope that the latest change, whether it is workload or PP or OP, will not be updated to the work object until it becomes false
```

Comment:
The summary part should describe what the proposal is. e.g. Do you propose a `work.karmada.io/paused` label?